IM_calculation/IM/im_calculation.py

```diff
@@ -481,6 +486,10 @@ def compute_measures_multiprocess(
                     for waveform in waveforms
                 ]
                 all_results.extend(p.starmap(compute_measure_single, array_params))
+                # replace the line above with the below for debugging
+                # for array_param in array_params:
+                #     result=compute_measure_single(*array_param)
+                #     all_results.append(result)
```

Comment:
Starmap debugs correctly in python3. It has both the starmap trace and the internal trace. Hence why we removed pool-wrapper. I don't think this is necessary to keep

Reply:
Sure - pycharm went crazy, but I am happy to remove.